src/components/NotificationList/NotificationList.module.css

```diff
@@ -0,0 +1,93 @@
+@import '../../design-tokens/mixins.css';
+/*------------------------------------*\
+    #NOTIFICATION LIST
+\*------------------------------------*/
+
+/**
+ * 1)  Notification list contains notification items, a count of unread notifications, and a button to mark all notifications as read
+ */
+.notification-list {
+    @mixin reset;
+    display: flex;
+    flex-direction: column;
+}
+/**
+* Individual notification list item
+*/
+.notification-list__item {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    padding: var(--eds-size-1);
+    background-color: var(--eds-color-brand-blue-100);/* TODO */
+    border: 1px solid var(--eds-color-neutral-200);/* TODO */
+}
+
+/* 
+* Notification list item content
+* 1) The wrapper for the text
+*/
+.notification-list__item--content {
+    color: var(--eds-color-neutral-600);/* TODO */
+    font-size: var(--eds-font-size-xs);
+    padding-left: var(--eds-size-3);
+    padding-right: var(--eds-size-3);
+    padding-bottom: var(--eds-size-1);
+    padding-top: var(--eds-size-1);
+}
+
+/*
+* Notification list item title
+* 1) Main text title of the notification
+*/
+.notification-list__item--title {
+    color: var(--eds-color-brand-blue-700);/* TODO */
+    font-size: var(--eds-font-size-sm)
+}
+
+/**
+* Notification list item link
+*  1) Anchor that wraps the title; linked to source of notification.
+*/
+.notification-list__item--link {
+    text-decoration: none;
+}
+
+/*
+* Notification list item subtitle
+* 1) Contains the created date (relative) and the originating source of the notification
+
+* Notification list item button
+* 1) Indicator that displays the item's read/not read status in
+* a <button> element; when clicked, it toggles between read/not
+* read statuses
+*/
+.notification-list__item--button {
+    border-radius: 4px;
+    height: 8px;
+    width: 8px;
+    background-color: var(--eds-color-brand-blue-700);/* TODO */
+    border: none;
+    padding: 0;
+    cursor: pointer;
+}
+
+/* 
+* Notification list item (is read)
+* 1) Class that applies neutral background to an item that is marked as read
+* 2) Title color is changed to neutral
+* 3) Button has fill removed
+*/
+
+.notification-list__item--is-read {
+    background-color: var(--eds-color-neutral-white);/* TODO */
+
+    .notification-list__item--title { /* 2 */
```

Comment:
CSS comment

Reply:
Let's also remove `item--` in front of that and consolidate this into the 1 ruleset above like this:

```
.notification-list__title {
    color: var(--eds-color-brand-blue-700);/* TODO */
    
    .notification-list__item--is-read & {
        color: var(--eds-color-neutral-600);/* TODO */
    }
}
```

Reply:
I removed the font size since we should be using a `Heading` within your `<a>` tag